package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@tinymce/tinymce-angular",
-  "version": "3.5.0",
+  "version": "3.5.1",
```

Comment:
Similar to the react PR, do we need to bump the version here as all you're updating is dev dependencies. Also if you do want to bump the version, then we'll need a changelog entry.

- [ ] Changelog or leave as-is please